plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9010,6 +9010,75 @@ public void testCollidingMixedCaseProperty()
         assertUpdate("DROP TABLE %s".formatted(tableName));
     }
 
+    @Test
+    public void testCommentViaSeparateCommentOnColumnStatement()
+    {
+        String table = "test_predicate_pushdown_" + randomNameSuffix();
+
+        assertUpdate("""
+                CREATE TABLE hive.tpch.%s (
+                   regular_column date,
+                   partition_column date
+                )
+                WITH (
+                   format = 'PARQUET',
+                   partitioned_by = ARRAY['partition_column']
+                )
+                """.formatted(table));
+
+        assertUpdate("COMMENT ON COLUMN %s.regular_column IS 'This is a regular column comment.'".formatted(table));
+        assertUpdate("COMMENT ON COLUMN %s.partition_column IS 'This is a partition column comment.'".formatted(table));
+        assertUpdate("COMMENT ON TABLE %s IS 'This is a table comment.'".formatted(table));
+
+        assertThat((String) getQueryRunner().execute("SHOW CREATE TABLE " + table).getOnlyValue())
```

Comment:
There's no need to validate the entire table definition. I would recommend using `BaseConnectorTest#getTableComment` method instead. 

 Same for another test method.